internal/schema/typeref.go

```diff
@@ -75,7 +77,39 @@ func (r *TypeRef) GetTags() string {
 	return jsonTag + "\"`"
 }
 
-// GetName returns a recusive lookup of the type name
+// GetTypeNameWithOverride returns the typeName, taking into consideration any TypeOverride specified in the PackageConfig.
+func (r *TypeRef) GetTypeNameWithOverride(pkgConfig *config.PackageConfig) (string, error) {
```

Comment:
Type is redundant here, as this is on TypeRef.  `GetNameWithOverride` ?

Reply:
Oh did I overlook an existing method?  Which part is redundant?

Reply:
The word 'Type' in the function name is redunant with the object it hangs off of.

**Type**.Get**Type**NameWithOverride()

Reply:
This got moved back to field.